### DISCUSSION TITLE
Can I subscribe to 'Copilot for Individuals' within my company?

### DISCUSSION THREAD

**Discussion by jinifor** [community/community#56832] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

We have an organization on GitHub, but not all of our developers are members and using it.

In this case, we're looking for clarity on whether the subscription policy is an issue when a developer who is not registered as a member of the organization uses a Copilot for Individuals.

We understand the difference between individuals and businesses.([About GitHub Copilot for Individuals](https://docs.github.com/en/copilot/overview-of-github-copilot/about-github-copilot-for-individuals), [About GitHub Copilot for Business](https://docs.github.com/en/copilot/overview-of-github-copilot/about-github-copilot-for-business))

However, I'm asking because I can't find anything that prohibits the use of Copilot for Individuals within an company.

**Comment by davecheney** [comment, marked as answer]
If you have paid for an individual copilot subscription it up to you or your employer to negotiate if they will permit you to use it while in their employment, we cannot provide any guidance.

However, if your employ would like to take out a Copilot subscription they can invite you to the organisation which will cancel your existing individual subscription and refund you the balance.

**Reply by jinifor** [reply]
@davecheney Perfect answer, Thanks.
